internal/backends/rlang/rlang.go

```diff
@@ -0,0 +1,138 @@
+package rlang
+
+import (
+	"fmt"
+	"regexp"
+	"github.com/ALANVF/rgo"
```

Comment:
When running `make full` locally on your branch, I encounter this error: 

```
cd cmd/upm && go build -ldflags "-X 'github.com/replit/upm/internal/cli.version=${VERSION:-development version}'"
# github.com/ALANVF/rgo
/root/go/pkg/mod/github.com/!a!l!a!n!v!f/rgo@v0.0.0-20200324171131-7f868d8e11f2/rgo.go:11:10: fatal error: R.h: No such file or directory
 #include <R.h>
          ^~~~~
compilation terminated.
make: *** [Makefile:12: cmd/upm/upm] Error 2
```

Building `rgo` depends on having installed the R headers. Currently you do that in `docker-install-languages.bash` (understandable!). Can you try installing `r-base-dev` in `docker-install-build.bash` and see if you're able to run `make full`?

Reply:
Yes I can try that. I had developed `rgo` on repl.it since I couldn't figure out how to use the R headers locally on windows, so I wasn't sure what dependencies were needed to make it work here

Reply:
Fixed now